manifests/plugin/container_gateway/params.pp

```diff
@@ -0,0 +1,11 @@
+# Default parameters for the Container Gateway smart proxy plugin
+# @api private
+class foreman_proxy::plugin::container_gateway::params {
+  $enabled    = true
+  $listen_on  = 'https'
+  $version    = undef
+  $pulp_endpoint  = "https://${facts['networking']['fqdn']}"
+  $pulp_ssl_client_cert = '/etc/foreman-proxy/foreman_ssl_cert.pem'
+  $pulp_ssl_client_key  = '/etc/foreman-proxy/foreman_ssl_key.pem'
```

Comment:
> Isn't this the same as $foreman_proxy::foreman_ssl_cert and key?

It is, I'll use `$foreman_proxy::foreman_ssl_cert & key` then.

We're using that cert and key to avoid handing out the Pulp 3 certs to every smart proxy.

> That also makes me think you should have a CA. 

I'll have to take a look at see what that entails installer-wise.

Reply:
Is `/etc/foreman-proxy/foreman_ssl_ca.pem` the CA that you were talking about?

Reply:
I'll make an issue on the container gateway to take a CA as a configurable parameter.